elide-async/src/main/java/com/yahoo/elide/async/export/TableExporter.java

```diff
@@ -26,44 +26,39 @@
 
 /**
  * Class for Table Export functionality.
+ * Parses a TableExport request and returns an Observable of PersistentResource.
  */
 @Slf4j
 public class TableExporter {
 
     private Elide elide;
```

Comment:
AsyncExecutorService seems to be a better place to keep the downloadURL instead of TableExporter. TableExporter will not have any use of downloadURL.